fv3/fv3_cap.F90

```diff
@@ -1266,6 +1356,7 @@ subroutine ModelAdvance_phase2(gcomp, rc)
           if (fieldCount > 0) then
             call ESMF_FieldBundleSMM(fcstFB(j), wrtFB(j,n_group),         &
                                      routehandle=routehandle(j, n_group), &
+                                     zeroregion=ESMF_REGION_SELECT,       &
```

Comment:
Is this required if the ESMF_FieldBundleRegridStore has a dstMaskValues mask? I saw in ESMF document (https://earthsystemmodeling.org/docs/nightly/v8.9.0b11/ESMF_refdoc/node5.html#SECTION050253300000000000000):

type(ESMF_Region_Flag), intent(in), optional :: zeroregion ! DEPRECATED ARGUMENT

Reply:
Yes, this flag is needed for moving nest. I replaced it with `zeroregionflag` in 4e82ca2